ui/src/org/pentaho/di/ui/spoon/messages/messages_en_US.properties

```diff
@@ -598,7 +598,10 @@ TransLog.Dialog.ErrorOpeningTransformation.Title=Error opening transformation
 Spoon.Tooltip.ShowJobHistoryView=Show or hide the logging history view of this job
 Spoon.Menu.Edit.EditKettleProperties=Edit the kettle.properties file
 Spoon.Candidate.HelpAboutText=Release Candidate 1
+
 Spoon.Tooltip.RunTranformation=Run this transformation or job
+Spoon.Run.Run=Run\t\t\t\tF9
+Spoon.Run.RunOptions=Run Options...\t\tF8
```

Comment:
As far as I know, there are no other places where we append the hotkeys to the UI messages. For consistency's sake, please remove the hotkey hints. (Great idea though, I'm just worried about portability and consistency)